src/context/mod.rs

```diff
@@ -379,19 +381,28 @@ impl Context {
                         capabilities: &*capabilities,
                     }),
                     Ok(Message::NextEvent(notify)) => {
-                        for event in window.poll_events() {
-                            // update the dimensions
-                            if let &glutin::Event::Resized(width, height) = &event {
-                                dimensions.0.store(width as usize, atomic::Ordering::Relaxed);
-                                dimensions.1.store(height as usize, atomic::Ordering::Relaxed);
-                            }
-
-                            // sending back
-                            notify.send(event).ok();
-                        }
+                        notifiers.push(notify)
                     },
                     Err(_) => break
                 }
+
+                for event in window.poll_events() {
```

Comment:
Are you sure? But I am continuously receiving all the events… I thought, this is run every frame.